inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/tasks/SelectionTask.java

```diff
@@ -130,29 +135,31 @@ public void run()
                     
                     if (factory == null) {
                         log.error("[{}][{}]: No recommender factory available for [{}]",
-                                user.getUsername(), r.getName(), r.getTool());
+                                userName, r.getName(), r.getTool());
+                        appEventPublisher.publishEvent(new TaskUpdateEvent(this, userName,
+                                TaskState.DONE, 1,
+                                String.format("No recommender factory available.", recommender)));
                         continue;
                     }
                     
                     if (!factory.accepts(recommender.getLayer(), recommender.getFeature())) {
                         log.info("[{}][{}]: Recommender configured with invalid layer or feature "
                                 + "- skipping recommender", user.getUsername(), r.getName());
+                        appEventPublisher.publishEvent(new TaskUpdateEvent(this, userName,
```

Comment:
Is it correct to send a `TaskState.DONE` here? There is just a `continue` here to proceed with the next recommender - this is not a `break`.